com.abisource.AbiWord.json

```diff
@@ -0,0 +1,218 @@
+                    "url" : "https://newcontinuum.dl.sourceforge.net/project/wvware/libwmf/0.2.8.4/libwmf-0.2.8.4.tar.gz",
+                    "sha256" : "5b345c69220545d003ad52bfd035d5d6f4f075e65204114a9e875e84895a7cf8"
+                }
+            ]
+        },
+        {
+            "name" : "wv",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.abisource.com/downloads/wv/1.2.9/wv-1.2.9.tar.gz",
+                    "sha256" : "4c730d3b325c0785450dd3a043eeb53e1518598c4f41f155558385dd2635c19d"
+                }
+            ]
+        },
+        {
+            "name" : "link-grammar",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.abisource.com/downloads/link-grammar/5.3.11/link-grammar-5.3.11.tar.gz",
+                    "sha256" : "73ca3ae82ec7ddbfce827eb7017a2953afea2cea854b4d7517139881e1170cd7"
+                }
+            ]
+        },
+        {
+            "name" : "lasem",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.gnome.org/sources/lasem/0.5/lasem-0.5.0.tar.xz",
+                    "sha256" : "d40aa6b82416dd9e17b62c0292aab5b2b00eb1d7c3b93cec85ac1bf1ac25b9b9"
+                }
+            ]
+        },
+        {
+            "name" : "goffice",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.gnome.org/sources/goffice/0.10/goffice-0.10.32.tar.xz",
+                    "sha256" : "02b37da9f54fb92725b973875d1d2da49b54f6486eb03648fd1ea58e4a297ac3"
+                }
+            ]
+        },
+        {
+            "name" : "libwpd",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://pilotfiber.dl.sourceforge.net/project/libwpd/libwpd/libwpd-0.9.9/libwpd-0.9.9.tar.xz",
+                    "sha256" : "b91df4ca0ef0f456be9531c9ee07cf6a5e50a16b52002abb99e8ebef13fac2b2"
+                }
+            ]
+        },
+        {
+            "name" : "libwpg",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://excellmedia.dl.sourceforge.net/project/libwpg/libwpg/libwpg-0.2.2/libwpg-0.2.2.tar.xz",
+                    "sha256" : "b49bfd77b9447534dee0d43bbdc48b4dcb09339605be21c01726243d5069a6cd"
+                }
+            ]
+        },
+        {
+            "name" : "psiconv",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://www.frodo.looijaard.name/system/files/software/psiconv/psiconv-0.9.9.tar.gz",
+                    "sha256" : "6d51fe79b502a1e277bea275a574ae2db5b1b9d7daef703a8fa3635ae02a8bb0"
+                }
+            ]
+        },
+        {
```

Comment:
is this supposed to be empty?

Reply:
I may I forgotten them. I'll deal with these optional dependencies later.